src/implementations/twig/components/file/file-translations.html.twig

```diff
@@ -3,6 +3,8 @@
 {#
   Parameters:
     - "toggle" (object) (default: {}): object of type Button
+    - "id" (string) (default: '')
+    - "title_id" (string) (default: '')
```

Comment:
as the title id is just `_id ~ '-title'`, do we need to pass it as a parameter everywhere?
It seems that it could be deduced from the global id when needed